packages/provider/tests/batch/commitments.test.ts

```diff
@@ -125,22 +131,33 @@ describe('BATCH TESTS', function () {
                 const commitmentIds: string[] = []
 
                 // Store 10 commitments in the local db
+                const completedAt = (await env.api.rpc.chain.getBlock()).block.header.number.toNumber()
+                const requestedAt = completedAt - 1
```

Comment:
not sure if i understand why requestedAt = completedAt - 1. what if it takes longer than 1 block to complete?

Reply:
This just a test to see if we can batch 50 commitments on-chain. The content of the commitments doesn't matter so I'm simply creating dummy ones here.